mod/research/lib/javascript/script_research.js.coffee

```diff
@@ -18,13 +17,19 @@ decko.slotReady (slot) ->
 $(document).ready ->
   $("#main:has(>#Research_Page.slot_machine-view)").addClass("pl-0 pr-0")
 
+  $($('ul.pointer-radio-list')[0]).find('input[type="radio"]').on "click", () ->
+    uncheckOrCheckUnknow false
+  $($('div.pointer-checkbox-list')[0]).find('input[type="checkbox"]').on "click", () ->
+    uncheckOrCheckUnknow false
+
   $('#card_subcards__values_content').on "keyup", () ->  
-    selector = '#card_subcards__values_subcards__Unknown_content'
     checked = $(this).val().toLowerCase() == 'unknown'
-    $(selector).prop 'checked', checked
+    uncheckOrCheckUnknow checked
 
    $('#card_subcards__values_subcards__Unknown_content').on "click", () -> 
```

Comment:
it would be nice to add a javascript-specific class for this (something starting with _)

Reply:
in what file would you add this class?